### ISSUE TITLE
Publish Foam to Vercel

### ISSUE THREAD

**Issue by taydr** [foambubble/foam#170]
I'd like a way to publish from my private Github repo to Vercel, but there's no clear documentation.

**Comment by mathieudutour** [comment]
We don't really have a documentation for it yet but here is an example of a jekyll website deployed to vercel: https://github.com/vercel/vercel/tree/master/examples/jekyll

**Comment by riccardoferretti** [comment]
Good find @mathieudutour, shall we add a recipe for this?

**Comment by mathieudutour** [comment]
Yeah seems like a good first step. As described here: https://github.com/foambubble/rfcs/pull/2 I'd like to support it a bit further but that will be down the line

**Comment by elgirafo** [comment]
> I'd like a way to publish from my private Github repo to Vercel, but there's no clear documentation.

this template made a good explanation of how to publish to to vercel
https://github.com/hikerpig/foam-template-gatsby-kb
